tests/dummy/lib/outdated/app/utils/floor-outdated.js

```diff
@@ -1,5 +1 @@
-import flooring from 'npm:flooring';
-
-export default function() {
-  return flooring();
-}
+export { default } from 'outdated/utils/floor-type';
```

Comment:
please coordinate before doing work on this branch. I am in the process of fixing these issues.